### PR TITLE
l10n: Spelling unification

### PR DESCRIPTION
**Pull request by Valdnet**
Spelling unification in Transifex.

Signed-off-by: Valdnet <47037905+Valdnet@users.noreply.github.com>

### CODE REVIEW

File: apps/updatenotification/src/components/UpdateNotification.vue [hunk 1]
```diff
@@ -194,7 +194,7 @@ export default {
 			}
 
 			if (this.appStoreFailed) {
-				return t('updatenotification', 'Could not connect to the appstore or the appstore returned no updates at all. Search manually for updates or make sure your server has access to the internet and can connect to the appstore.')
+				return t('updatenotification', 'Could not connect to the App Store or returned no updates at all. Search manually for updates or make sure your server has access to the internet and can connect to the App Store.')
```
Comment (rakekniven) [inline apps/updatenotification/src/components/UpdateNotification.vue:197]: 'Could not connect to the App Store or no updates have been returned at all.'

**Review by rakekniven — CHANGES_REQUESTED** [review]

**Review by rakekniven — APPROVED** [review]

**Comment by MorrisJobke** [comment]
@jospoortvliet Fine with you?

**Review by MorrisJobke — APPROVED** [review]

**Review by gary-kim — APPROVED** [review]

**Comment by Valdnet** [comment]
@nickvergessen Can you merge it 😉?

**Comment by Valdnet** [comment]
@nickvergessen So what are we doing with it?

**Pull request merged**